radikaledward1/dating-web-app
# Test Change

Description:
Test Change. it Adds the "Typescript" word at the final of the main page title.

Ticket:
DTA-001

How to Test:
- Clone the branch
- Run the app
- Verify tha the word "Typescript" is displayed.
